app/controllers/import_controller.rb

```diff
@@ -38,108 +46,139 @@ def import
 
     # Load and parse patient import excel
     begin
-      xlsx = Roo::Excelx.new(params[:file].tempfile.path, file_warning: :ignore)
-      validate_headers(format, xlsx.sheet(0).row(1))
-      raise ValidationError.new('File must contain at least one monitoree to import', 2) if xlsx.sheet(0).last_row < 2
-      raise ValidationError.new('Please limit each import to 1000 monitorees.', 1000) if xlsx.sheet(0).last_row > 1001
+      if format == :epix
+        ext = File.extname(params[:file].tempfile.path)
+        raise FileFormatError.new('File format must be .csv', ext) unless ext == '.csv'
+
+        csv = CSV.parse(File.read(params[:file].tempfile.path), headers: true)
+        validate_headers(format, csv.headers)
+      else
+        xlsx = Roo::Excelx.new(params[:file].tempfile.path, file_warning: :ignore)
+        validate_headers(format, xlsx.sheet(0).row(1))
+      end
+
+      num_rows = format == :epix ? csv.length : xlsx.sheet(0).last_row
+      raise ValidationError.new('File must contain at least one monitoree to import', 2) if num_rows < 2
+      raise ValidationError.new('Please limit each import to 1000 monitorees.', 1000) if num_rows > 1001
 
-      # define patients for duplicate detection here to avoid duplicate queries
+      # Define patients for duplicate detection here to avoid duplicate queries
       patients_for_duplicate_detection = current_user.viewable_patients
 
-      xlsx.sheet(0).each_with_index do |row, row_ind|
-        next if row_ind.zero? # Skip headers
+      records = format == :sara_alert_format ? xlsx.sheet(0) : csv
```

Comment:
Very minor but it may make sense to reverse this check to see if `format == :epix` just for consistency with the above.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/884/commits/0792f6104ae4af0333091eb76fb21a3418b3e5bd#diff-ee31e076a80cc9f48da0394c0cb066d59bf2f8c68f68eb62cb5119d01712d90eR67